gateway/gateway.go

```diff
@@ -418,12 +419,6 @@ func (gateway *HandleT) getSourceTagFromWriteKey(writeKey string) string {
 //
 // Finally sends responses(error) if any back to the webRequests over their `done` channels
 func (gateway *HandleT) userWebRequestWorkerProcess(userWebRequestWorker *userWebRequestWorkerT) {
-	if userWebRequestWorker == nil {
```

Comment:
Why panics removed?

Reply:
Yeah I introduced them recently for debugging.

Reply:
👍